ethereum/deployments.ts

```diff
@@ -41,7 +43,9 @@ export const deployments: RigsDeployments = {
     allowlistTable: "rigs_allowlist_42161_14",
     partsTable: "parts_42161_7",
     layersTable: "layers_42161_8",
+    rigsTable: "",
     attributesTable: "rig_attributes_42161_15",
+    dealsTable: "",
```

Comment:
New entries for the new rigs and deals tables. We'll need to fill these in for mainnet once we create those tables.